resolwe_bio/docker_images/latex/Dockerfile

```diff
@@ -4,3 +4,7 @@ MAINTAINER Resolwe Bioinformatics authors https://github.com/genialis/resolwe-bi
 
 # Force matplotlib backend to a headless one.
 ENV MPLBACKEND Agg
+
+# Additional files:
+COPY files/template_amplicon_report_multi_sample.tex /opt/resolwebio/template_amplicon_report_multi_sample.tex
```

Comment:
Please collapse the number of file copy operations to reduce the number of layers (e.g. copy the whole directory into `/assets` or something). I would also rename `files` to `assets`.

Reply:
Good point, fixed.